util/fipstools/acvp/modulewrapper/modulewrapper.cc

```diff
@@ -1599,27 +1611,35 @@ static bool ECDSASigVer(const Span<const uint8_t> args[], ReplyCallback write_re
   ECDSA_SIG sig;
   sig.r = r.get();
   sig.s = s.get();
-
-  uint8_t digest[EVP_MAX_MD_SIZE];
-  unsigned digest_len;
-  if (!key || !hash ||
-      !EVP_Digest(msg.data(), msg.size(), digest, &digest_len, hash,
-                  /*impl=*/nullptr)) {
+  uint8_t *der;
+  size_t der_len;
+  if (!key || !hash || !ECDSA_SIG_to_bytes(&der, &der_len, &sig)) {
     return false;
   }
-
+  // Let |delete_der| manage the release of |der|.
+  bssl::UniquePtr<uint8_t> delete_der(der);
```

Comment:
Is it possible to wrap `*der` in the unique pointer directly and use `der.get()` to call it afterwards, without using this?

Reply:
Not work.
```
../util/fipstools/acvp/modulewrapper/modulewrapper.cc:1616:62: error: lvalue required as unary ‘&’ operand
   bssl::UniquePtr<uint8_t> delete_der;
   if (!key || !hash || !ECDSA_SIG_to_bytes(&(delete_der.get()), &der_len, &sig)) {
                                                              ^
```